main.tex

```diff
@@ -89,6 +89,7 @@
 \showlistofprograms
 \showlistofappendices
 \showlistofacronyms
+\showlistofsymbols
```

Comment:
This all looks good! Since most people don't use the list of symbols (as you can tell from the typos you corrected, thank you!!), would you mind flipping this to `\hidelistofsymbols` as the default? Then I'll approve and merge. Thanks!

Reply:
Changed!